packages/cli/src/cms/commands/push-status.ts

```diff
@@ -31,123 +51,222 @@ export async function handlePushStatus(argv: PushStatusOptions, config: Config)
   const orgId = organization || config.organization;
 
   if (!orgId) {
-    return exitWithError(
+    exitWithError(
       `No organization provided, please use --organization option or specify the 'organization' field in the config file.`
     );
+    return;
   }
 
   const domain = argv.domain || getDomain();
-
-  if (!domain) {
```

Comment:
Why have you removed this check? Isn't it still required (at least a fallback)?

Reply:
@tatomyr This check does not make any sense since `getDomain()` has a default value - `https://app.cloud.redocly.com`